pom.xml

```diff
@@ -11,7 +11,7 @@
     <relativePath/> <!-- lookup parent from repository -->
   </parent>
 
-  <groupId>org.eightwelve</groupId>
+  <groupId>co.eightwelve</groupId>
   <artifactId>cdt-be</artifactId>
```

Comment:
co.eightwelve.cdt:cdt-be

Taken from spring cloud pom:
<groupId>org.springframework.cloud</groupId>
<artifactId>spring-cloud-netflix</artifactId